www/install/sql/centreon/Update-DB-19.10.2.sql

```diff
@@ -0,0 +1 @@
+UPDATE topology SET topology_url = NULL, topology_url_opt = NULL WHERE topology_page = 501;
```

Comment:
shouldn't this should be added in the last update file ?

Reply:
File deleted